src/backend/marsha/core/tests/test_services_video_recording.py

```diff
@@ -122,6 +122,7 @@ def test_services_video_recording_stop_recording_not_already_started(self):
 
         self.assertEqual(str(context.exception), "Video recording is not started.")
 
+    @override_settings(LIVE_SEGMENT_DURATION_SECONDS=10)
```

Comment:
Why do you need to override the value here ?

Reply:
I did it to control its value.